core/controllers/subtopic_viewer_test.py

```diff
@@ -74,7 +74,8 @@ def setUp(self):
                     'en': {
                         'filename': 'test.mp3',
                         'file_size_bytes': 100,
-                        'needs_update': False
+                        'needs_update': False,
+                        'duration': 0.34234
```

Comment:
I found that the voiceover class is used in subtopics and topic but I don't think you have to write the migration for the same as there isn't any audio voiceover for topics and subtopics on the server. 

@aks681, Can you please confirm?

Reply:
Yeah, there are no audio recordings for topics or subtopics on the server.

Reply:
Yeah just needed it for the backend tests.